src/main.ts

```diff
@@ -448,6 +448,12 @@ export class Purchases {
       throw new Error("You cannot use paywalls yet, they are coming soon!");
```

Comment:
Would it be worth changing this error to something like
```suggestion
      throw new Error("No paywall found for this offering. Please make sure you've added a paywall in the RevenueCat dashboard.");
```

Not directly related to this PR though.